Test/triggers/loop-detection-messages--unit-tests.dfy.expect

```diff
@@ -1,28 +1,28 @@
 loop-detection-messages--unit-tests.dfy(11,9): Info: Selected triggers: {f(f(i))}
- Rejected triggers: {f(i)} (may loop with "f(f(i))") Trigger position: loop-detection-messages--unit-tests.dfy(11,36)
-loop-detection-messages--unit-tests.dfy(12,9): Info: Selected triggers: {f(_t#0), f(i)} Trigger position: loop-detection-messages--unit-tests.dfy(12,36)
-loop-detection-messages--unit-tests.dfy(13,9): Info: Selected triggers: {f(_t#0), f(i)} Trigger position: loop-detection-messages--unit-tests.dfy(13,52)
+ Rejected triggers: {f(i)} (may loop with "f(f(i))")
+loop-detection-messages--unit-tests.dfy(12,9): Info: Selected triggers: {f(_t#0), f(i)}
+loop-detection-messages--unit-tests.dfy(13,9): Info: Selected triggers: {f(_t#0), f(i)}
 loop-detection-messages--unit-tests.dfy(15,9): Info: For expression "false ==> f(i) == f(_t#0)":
-   Selected triggers: {f(_t#0), f(i)} Trigger position: loop-detection-messages--unit-tests.dfy(15,36)
+   Selected triggers: {f(_t#0), f(i)} in sub-expression at loop-detection-messages--unit-tests.dfy(15,36)
```

Comment:
I think there's still a problem here: the trigger is made up of two subexpressions, `f(_t#0)` and `f(i)`; which position are we reporting?

Reply:
We are reporting the position of the token "==", which is the token held by the binaryExpression "f(i) == f(_t#0)" that cannot be further divided.

Reply:
Note that when expressions are going to report ranges instead of tokens, we might be able to display the range of the expression instead of the token.

Reply:
I *think* I understand what this position means, but I expect it will confuse users. Do you think we need to display it?

Reply:
Glad that we both think we understand what these positions mean.
The alternative would be to remove it, but 1) we won't think of putting it back when we will have ranges and 2) it makes sense for someone who see that there are several "For expression" headers to understand where is that expression that it considered.

We don't display these trigger information by default anyway (it requires a special command line, and in the IDE it's a popup on the comprehensions that we probably want to simplify anyway), only those who care about selected triggers will want to learn about them.
These positions are also used in Boogie to report which condition in a for loop is not satisfied, so that's the single good reason why we want to keep them around. Sounds fair?

Reply:
OK, sounds fair